index.html

```diff
@@ -282,8 +316,16 @@ <h2>About me</h2>
       </form>
 
       <div class="rectangle">
-        <img src="./images/projects/footericons/Rectangle 42.svg" alt="" />
+        <img
+          class="disappear"
+          src="./images/projects/footericons/Rectangle 42.svg"
+          alt="horizontal"
+        />
       </div>
+      <div class="crossimage1">
+        <img src="./images/desky/crossimage.svg" alt="crossimage" />
+      </div>
+
       <ul>
         <li>
           <a href="#"
```

Comment:
- [ ] Kindly make sure you fix the spacing at the bottom of yout footer to match the Figma desgn as right now you a big space compared to the one from the template.
![Screenshot from 2023-06-08 19-23-12](https://github.com/mj-e-boc/M-J-Portfolio/assets/83042742/db793454-d96d-4088-9d9f-985b58d978f5)